### DISCUSSION TITLE
self hosted + keycloak

### DISCUSSION THREAD

**Discussion by syniex** [supabase/supabase#8994] [Questions]
Thanks you an awesome project.

I tried to setup self hosted (Docker)  Supabase and connect it to my Keycloak.

I couldn't get pass the error "Could not verify token" in the GoTrue log.
Does anyone tried to setup keycloak with supabase on self hosted environment?

**Comment by buff** [comment]
Having a similar issue running a nuxt 3 project.
I have also seen that the re-routing to the keycloak instance does not always happen.
